csharp/src/Apache.Arrow/Ipc/ArrowFileReaderImplementation.cs

```diff
@@ -110,7 +110,7 @@ protected override void ReadSchema()
 
             ArrayPool<byte>.Shared.RentReturn(footerLength, (buffer) =>
             {
-                _footerStartPostion = (int)GetFooterLengthPosition() - footerLength;
+                _footerStartPostion = GetFooterLengthPosition() - footerLength;
```

Comment:
We also need to fix the same code in `ReadSchemaAsync()`. It has a similar cast.